packages/idempotency/src/PersistenceLayer.ts

```diff
@@ -0,0 +1,28 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+import { IdempotencyConfig } from './IdempotencyConfig';
+import { PersistenceLayerInterface } from './PersistenceLayerInterface';
+
+abstract class PersistenceLayer implements PersistenceLayerInterface {
+  public configure(_config: IdempotencyConfig): void {}
```

Comment:
I've been distracted so I may have missed the reasoning, but why isn't this the constructor?

Reply:
We followed what the Python version had, the persistence layer does have its own constructor defined that saves other things on the constructor. I don't have the constructor here currently because many of those objects are related to config that is outside the scope of what we plan to deliver for the first set of functionality

Reply:
https://github.com/awslabs/aws-lambda-powertools-python/blob/160feae22e24708bc5bb4cf979a0c05c99db87a4/aws_lambda_powertools/utilities/idempotency/persistence/base.py#L116